### PR TITLE
[Wait for #1088, #1132] [Ini] Remove IniTestWrapper @open sesame 04/22 18:26

### PR DESCRIPTION
**Pull request by zhoonit**
- [Ini] Remove IniTestWrapper 

```
This patch removes initestwrapper in favor of nntrianer::iniwrapper

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

File: test/ccapi/unittest_ccapi.cpp [hunk 1]
```diff
@@ -152,36 +153,38 @@ TEST(ccapi_dataset, construct_02_p) {
   EXPECT_NO_THROW(ml::train::createDataset(ml::train::DatasetType::FILE));
 }
```
Comment (kparichay) [inline test/ccapi/unittest_ccapi.cpp:155]: how about `using nntrainer::IniSection?`

Comment (zhoonit) [reply test/ccapi/unittest_ccapi.cpp:155]: Not a big deal though, I wonder if i should invest more time to change. I think qualified name works just fine as well.

Comment (kparichay) [reply test/ccapi/unittest_ccapi.cpp:155]: Either is fine.

**Review by kparichay — APPROVED** [review]
LGTM

**Review by lhs8928 — APPROVED** [review]
LGTM

**Pull request merged**